src/index.html

```diff
@@ -17,6 +17,10 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <main class="watch watch--speed-up">
```

Comment:
Consider fixing class to pass tests 
```suggestion
    <main class="stopwatch stopwatch--speed-up">
```